### PR TITLE
feat(reports): 050724 contributors report

### PR DESCRIPTION
**Pull request by SteBaum**
Contributors' report of 5th of July 2024.

### CODE REVIEW

File: content/reports/240705-contributors.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+type: contributors
+description: |
+  PR tdp-collection#850 and the use of Alluxio
+---
+
+# Notes from July 5th, 2024
+
+PR tdp-collection#850 and the use of Alluxio
```
Comment (wdavidw) [inline content/reports/240705-contributors.md:9]: dot at the end of sentences

File: content/reports/240705-contributors.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- [jupyterhub#1](https://github.com/TOSIT-IO/jupyterhub/pull/1): status of "Add Readme". PR merged. PR merged.
+- [hue#2](https://github.com/TOSIT-IO/hue/pull/2): status of "refactor: build with manylinux2014 image". PR merged. However, find another name for the branch and for the release.
+- [tdp-website#101](https://github.com/TOSIT-IO/tdp-website/pull/101): status of "feat(reports): 240628 contributors report". PR merged.
+
+## Open topics
+
+Added patches for Hive 3 which gives us the release 3.1.3-2.3.
+
+Work on the integration of PrestoSQL has started. However, Alluxio might be used instead as it will work more efficiently with Hive as it uses a cache and will run with Yarn, has better capacity scheduling for multiple users as well as enables to manage S3 permissions with Ranger. To be further discussed, but work has started on it.
```
Comment (wdavidw) [inline content/reports/240705-contributors.md:26]: Split the second sentence into 3 sentences. "However, Alluxio might be used instead. Its usage by Hive is of particular interest, leveraging its cache layer and YARN integration. Additionally, we are interested by its capacity scheduling for multiple users as well as its ability to manage S3 permissions with Ranger."

**Review by wdavidw — CHANGES_REQUESTED** [review]

**Review by wdavidw — APPROVED** [review]

**Comment by omazain** [comment]
Je suis absent du 28 juin 2024 au 19 juillet 2024 inclus. En cas d'urgence, merci de contacter Madame Florence THILLAY

**Comment by abeguec** [comment]
Je suis absente du 01 juillet 2024 au 17 juillet 2024 inclus. Si besoin, vous pouvez contacter Mme Florence Thillay

**Comment by nviault-dgfip** [comment]
Je suis absent du 01 juillet 2024 au 12 juillet 2024 inclus.

**Pull request merged**